local_ads/local_ads_tests/local_ads_helpers_tests.cpp

```diff
@@ -0,0 +1,71 @@
+#include "testing/testing.hpp"
+
+#include "local_ads/local_ads_helpers.hpp"
+
+#include "coding/file_name_utils.hpp"
+
+#include "platform/platform_tests_support/scoped_file.hpp"
+
+UNIT_TEST(LocalAdsHelpers_Read_Write_Country_Name)
+{
+  platform::tests_support::ScopedFile testFile("la_tests.dat");
```

Comment:
Please, put
```c++
using namespace std;
```
And get rid of these "std::" prefixes. Feel free to add usings for local_ads to.